pkg/cmd/root/root.go

```diff
@@ -92,9 +92,14 @@ func NewCmdRoot(f *cmdutil.Factory, version, buildDate string) *cobra.Command {
 
 	// Help topics
 	cmd.AddCommand(NewHelpTopic("environment"))
+	referenceCmd := NewHelpTopic("reference")
+	referenceCmd.SetHelpFunc(referenceHelpFn(f.IOStreams))
+	cmd.AddCommand(referenceCmd)
 
 	cmdutil.DisableAuthCheck(cmd)
 
+	// this needs to appear last:
+	referenceCmd.Long = referenceLong(cmd)
```

Comment:
I dislike this approach but don't see value in further back and forthing on this given that we only have two help topics